Demo/build.gradle

```diff
@@ -70,6 +70,7 @@ dependencies {
     implementation deps.composeRuntimeLivedata
     implementation deps.composeFoundation
     implementation deps.composeMaterial
+    implementation "androidx.constraintlayout:constraintlayout-compose:1.0.0-alpha07"
```

Comment:
can you match this line with the dependencies style?

Reply:
yep, i forgot hehe